typings/typescript.d.ts

```diff
@@ -0,0 +1,11 @@
+import "typescript";
+
+declare module "typescript" {
+	// internal TS APIs
+
+	interface Symbol {
+		readonly links?: {
+			readonly checkFlags: ModifierFlags;
```

Comment:
Yeah I can't find any other way to do this but don't _love_ relying on TS internals 😬 ... hmm. Do you think you could file an issue on TypeScript @RebeccaStevens asking to make this or an equivalent public?

Reply:
https://github.com/microsoft/TypeScript/issues/58656